app/src/main/java/pm/gnosis/heimdall/reporting/impl/FabricEventTracker.kt

```diff
@@ -1,19 +1,55 @@
 package pm.gnosis.heimdall.reporting.impl
 
+import com.crashlytics.android.Crashlytics
 import com.crashlytics.android.answers.Answers
 import com.crashlytics.android.answers.ContentViewEvent
+import com.crashlytics.android.answers.CustomEvent
 import io.reactivex.Single
+import pm.gnosis.heimdall.reporting.Event
 import pm.gnosis.heimdall.reporting.EventTracker
+import timber.log.Timber
 import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
 class FabricEventTracker @Inject constructor(
         private val fabricCore: FabricCore
 ) : EventTracker {
+
+    private val answers = Answers.getInstance()
```

Comment:
Can this be provided with Dagger?

Reply:
I wouldn't, I could probably access it via the fabric core ... but it's the same as with the Crashlytics singleton

Reply:
Yea. It's just to try to do more DI.